src/common/lib/util/logger.ts

```diff
@@ -117,7 +117,7 @@ class Logger {
 
   private logAction(level: LogLevels, action: string, message?: string) {
     if (this.shouldLog(level)) {
-      (level === LogLevels.Error ? this.logErrorHandler : this.logHandler)('Ably: ' + action + ': ' + message);
+      (level === LogLevels.Error ? this.logErrorHandler : this.logHandler)('Ably: ' + action + ': ' + message, level);
```

Comment:
I think we also need to slightly change `getDefaultLoggers` function and ignore second argument for them. Otherwise I think default console logger will print log level at the end of log

Reply:
I checked this - it doesn't seem to log them!

Reply:
OK, you won 😄  because of weird `logTimestamps` that is always `true`. It never use default `console.log` as `logHandler` and always wrap it in high-level function that ignores all arguments except first. But if somebody make it `false` you'll see everything will fall apart. I would change `getHandler` function from:

```ts
function getHandler(logger: Function): Function {
  return Platform.Config.logTimestamps
    ? ...
    : logger;
}
```
to 
```ts
function getHandler(logger: Function): Function {
  return Platform.Config.logTimestamps
    ? ...
    : function (msg: unknown) {
        logger(msg);
      };
}
```

Reply:
:D done!